ansible/terraform/shakenfist/setup.tf

```diff
@@ -0,0 +1,135 @@
+variable "namespace" {
+  description = "The Shaken Fist namespace to create"
+}
+
+variable "system_key" {
+  description = "The key for the Shaken Fist system namespace"
+}
+
+variable "ssh_key" {
+  description = "SSH public key text for ansible instance authentication"
+}
+
+variable "uniqifier" {
+  description = "A unique string to prefix hostnames with"
+}
+
+// Create new namespace using Shaken Fist "system" privilege.
+provider "shakenfist" {
+  alias      = "system"
+  server_url = "http://localhost:13000"
```

Comment:
Should this be configurable?

Reply:
I think I'd rather take these from environment variables. You had a patch to do that, but I think it was for the rest client? Maybe I should route the environment variables in via a var here. For now I am just trying to get this to work _at_all_.

Reply:
Let's punt on this for now until we're closer to it working for real?